web/generate_site/generate.py

```diff
@@ -19,15 +19,36 @@
 
 ##### Template configuration
 conf = {
-    "ORG": df_org.to_dict('records'),
+    "ORG": df_org.to_dict("records"),
 }
 
-templates = ["about", "index"]
-environment = Environment(loader=FileSystemLoader("templates/"))
+templates = {"about":
+    {
+        "og_title": "PyLadiesCon 2023 - About Us",
+        "og_description": "About PyLadiesCon.",
+        "og_type": "article",
+        "og_url": "https://conference.pyladies.com/about.html",
+        "og_image_url": "https://conference.pyladies.com/img/icon.png",
```

Comment:
```suggestion
        "og_image_url": "https://conference.pyladies.com/img/icon.png",
        "og_image_alt": "PyLadiesCong logo',
```
Let's add alt text to our images to make these accessible to screen readers and other assistive tech